src/modules/users/controller/usersController.ts

```diff
@@ -27,4 +28,16 @@ const getUser = async (req: Request, res: Response) => {
     }
 }
 
-export default { getUsers, getUser }
\ No newline at end of file
+const logout = async (req: Request, res: Response) => {
+    const authHeader = req.headers.authorization;
+    if (!authHeader || !authHeader.startsWith('bearer ')) {
+        return res.status(401).json({ message: 'Invalid authorization header' });
```

Comment:
I think here you were supposed to follow same response format as other functions 
something like this
`responseUtil.handleError(UNAUTHORIZED, "Invalid authorization header");
 return responseUtil.response(res);`